BACKLOG.md

```diff
@@ -0,0 +1,38 @@
+BACKLOG
+=======
+
+General refactoring
+-------------------
+There are 3 representation of a tree.
```

Comment:
5 actually (at least): TTree, LTree, MetaTTree, CostTree, GFAbsTree

GFAbsTree is used by PGF, but should only be used in an PGF interface module (e.g., Tree/Internal.hs). Currently it's also used in Main.hs